htdocs/admin/delais.php

```diff
@@ -423,7 +431,7 @@
 			print ' &lt;= '.$level3;
 			print '</div><div class="inline-block" style="padding-right: 20px">';
 			print img_weather($text, 4, $options);
-			print ' &gt; '.$level3;
+			print ' &gt; '.$level4;
```

Comment:
Same here, the last level is when value is > level3 value (previous is when it is <).

Reply:
@eldy i revert my commit for this file